### PR TITLE
Minor fixes to make tests work with 'hardware' profile

### PR DESCRIPTION
**Pull request by rtretyak**

### CODE REVIEW

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/isl/IslMovementSpec.groovy [hunk 1]
```diff
@@ -38,12 +39,12 @@ class IslMovementSpec extends BaseSpecification {
         def isl = topology.islsForActiveSwitches.find {
             it.getAswitch()?.inPort && it.getAswitch()?.outPort
         }
-        assert isl
 
         and: "A non-connected a-switch link"
         def notConnectedIsl = topology.notConnectedIsls.find {
             it.srcSwitch != isl.srcSwitch && it.srcSwitch != isl.dstSwitch
```
Comment (npliashechnikov) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/isl/IslMovementSpec.groovy]: Maybe it'd make sense to use safe navigation here if you don't assert ISL anymore?

Comment (rtretyak) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/isl/IslMovementSpec.groovy]: reworked this place

File: services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/BaseSpecification.groovy [hunk 2]
```diff
@@ -83,7 +83,9 @@ class BaseSpecification extends SpringSpecification implements SetupOnce {
                     .collect { sw -> topologyDefinition.getSwitches().find { it.dpId == sw.switchId } }
 
             nonVirtualSwitches.findAll {
-                it.ofVersion != "OF_12" && !floodlight.getMeters(it.dpId).isEmpty()
+                it.ofVersion != "OF_12" && !floodlight.getMeters(it.dpId).findAll {
+                    it.key > 10 // ids that <=10 are reserved for 'default' meters for default rules
```
Comment (surabujin) [inline services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/BaseSpecification.groovy]: Now we have 15 reserved `id` for `default` rules.

Comment (rtretyak) [reply services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/BaseSpecification.groovy]: fxd

**Pull request merged**